pages/articles/[filename].tsx

```diff
@@ -0,0 +1,138 @@
+import { Breadcrumbs } from "@/blocks/breadcrumbs";
+import { componentRenderer } from "@/blocks/mdxComponentRenderer";
+import { TechUpgrade } from "@/blocks/techUpgrade";
+import { Blocks } from "@/components/blocks-renderer";
+import { Layout } from "@/components/layout";
+import MicroservicesPanel from "@/components/microservices/microservicesPanel";
+import { Section } from "@/components/util/section";
+import { SEO } from "@/components/util/seo";
+import { RecaptchaContext } from "@/context/RecaptchaContext";
+import { removeExtension } from "@/services/client/utils.service";
+import client from "@/tina/client";
+import classNames from "classnames";
+import { InferGetStaticPropsType } from "next";
+import { tinaField, useTina } from "tinacms/dist/react";
+import { TinaMarkdown } from "tinacms/dist/rich-text";
+
+export default function ArticlesPage(
+  props: InferGetStaticPropsType<typeof getStaticProps>
+) {
+  const { data } = useTina({
+    data: props.data,
+    query: props.query,
+    variables: props.variables,
+  });
+
+  return (
+    <RecaptchaContext.Provider
```

Comment:
what do we need recaptcha for? I thought this was removed from the project?

Reply:
removed